src/azure-cli/azure/cli/command_modules/backup/_client_factory.py

```diff
@@ -50,6 +50,10 @@ def vaults_cf(cli_ctx, *_):
     return _common_client_factory(cli_ctx).vaults
 
 
+# def registered_identities_cf(cli_ctx, *_):
+#    return _common_client_factory(cli_ctx).registered_identities
+
+
```

Comment:
Why do wee need to add this block?

Reply:
In PR description third point I have added this. I've commented some code as we were not able to test it properly. We will plan to take this with the next release so I've just commented that code.